README.md

```diff
@@ -560,3 +561,344 @@ notebook
+      "error": "Cannot find compatible environment: no compatible Local environment with Python version 3.9.5",
+      "code": 1,
+      "finished": true,
+      "result": {
+        "data": "An error occurred while building the content",
+        "type": "build-failed-error"
+      },
+      "id": "pZoqfBoi6BgpKde5"
+    },
+    "dashboard_url": "https://connect.example.org:3939/connect/#/apps/4ffc819c-065c-420c-88eb-332db1133317",
+    "name": "logs-api-python",
+    "title": "logs-api-python",
+    "content_url": "https://connect.example.org:3939/content/4ffc819c-065c-420c-88eb-332db1133317/",
+    "bundle_id": "141",
+    "rsconnect_last_build_time": "2021-12-14T18:07:16Z",
+    "created_time": "2021-07-19T19:17:32Z",
+    "app_mode": "python-api"
+  }
+]
+
+$ rsconnect content build logs --guid 4ffc819c-065c-420c-88eb-332db1133317
+[INFO] 2021-12-14T13:09:27-0500 Initializing ContentBuildStore for https://connect.example.org:3939
+Building Python API...
+Cannot find compatible environment: no compatible Local environment with Python version 3.9.5
+Task failed. Task exited with status 1.
+```
+
+
+## Common Usage Examples
+
+### Searching for content
+
+The following are some examples of how publishers might use the
+`rsconnect content search` subcommand to find content on RStudio Connect.
+By default, the `rsconnect content search` command will return metadata for ALL
+of the content on a RStudio Connect server, both published and unpublished content.
+
+```bash
+# return only published content
+$ rsconnect content search --published
+
+# return only unpublished content
+$ rsconnect content search --unpublished
+
+# return published content where the python version is at least 3.9.0
+$ rsconnect content search --published --py-version ">=3.9.0"
+
+# return published content where the R version is exactly 3.6.3
+$ rsconnect content search --published --r-version "==3.6.3"
+
+# return published content where the content type is a static RMD
+$ rsconnect content search --content-type rmd-static
+
+# return published content where the content type is either shiny OR fast-api
+$ rsconnect content search --content-type shiny --content-type python-fastapi
+
+# return all content, published or unpublished, where the title contains the text "Stock Report"
+$ rsconnect content search --title-contains "Stock Report"
+
+# return published content, results are ordered by when the content was last deployed
+$ rsconnect content search --published --order-by last_deployed
+
+# return published content, results are ordered by when the content was created
+$ rsconnect content search --published --order-by created
+```
+
+### Finding r and python versions
+
+One common use for the `search` command might be to find the versions of
+r and python that are currently in use on your RStudio Connect server before a migration.
+This information can be used later to help define a set of content images
+to be used with remote content execution on Kubernetes.
```

Comment:
Not sure if it's early to mention Kubernetes in the public docs since it isn't supported by Connect just yet?